doc/manual/getting_started_mgmt.rst

```diff
@@ -46,6 +46,30 @@ The dashboard should display the list of experiments from the repository folder
 
 .. note:: Multiple clients may be connected at the same time, possibly on different machines, and will be synchronized. See the ``-s`` option of ``artiq_dashboard`` and the ``--bind`` option of ``artiq_master`` to use the network. Both IPv4 and IPv6 are supported.
 
+Using a monitor/injection proxy for core device
+-----------------------------------------------
+
+If you do not have direct access to the subnet of control plane, e.g. not having symmetrical access to core device
+and master due to running in heterogeneous network architecture like one device handles all network communications by
+using NAT, or if you want to virtualize the monitor/injection controls to better handle dynamic core device transitions,
+then you could use the monitor/injection proxy after the master server was initiated: ::
+
+    $ aqctl_proxy_moninj --master-addr <master address> --port <proxy port> --bind <bind address>
+
+And add an entry of moninj to device database: ::
+
+    device_db['moninj'] = {
+        "type": "controller",
+        "host": "::1",
+        "port": "2383",
+        "master_addr": "localhost",
```

Comment:
**_Maybe_** replace `localhost` with `::1`, just to make it less confusing.

Reply:
Actually, you mentioned the possibility of connecting other network already, maybe you should instead parametrize the entire `moninj` entry. A default `moninj` entry is already generated from `artiq_ddb_template`.

Reply:
This is needed for existing users who are upgrading from ARTIQ 6 Iirc